Description/en-US.md

```diff
@@ -1,75 +1,64 @@
 # English (US)
+<!-- Use your language name, in your language as the title. Append the country in parentheses, if your language has regional variants. -->
 
-A Fabric-based modpack for Minecraft that focuses on performance and graphics optimizations. 
+A Fabric-based Minecraft modpack focusing on performance and graphics enhancements.
 
-[![Current review](https://img.youtube.com/vi/bb8G9X5Q_4I/hqdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
+<a href="https://www.youtube.com/watch?v=bb8G9X5Q_4I"><img src="https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg" alt="Current review" height="140"/> - A YouTube showcase by Rexus</a>
```

Comment:
I've changed all images to be smaller on upstream, so I suggest reverting this change.